ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -73,25 +73,12 @@ public enum DrawerPresentationBackground: Int {
     }
 }
 
-// MARK: - Drawer Color
-
-public extension Colors {
-    private struct Drawer {
-        static var background = UIColor(light: surfacePrimary, dark: surfaceSecondary)
-        static var popoverBackground = UIColor(light: surfacePrimary, dark: surfaceQuaternary)
-    }
-
-    // Objective-C support
-    @objc static var drawerBackground: UIColor { return Drawer.background }
-    @objc static var popoverBackground: UIColor { return Drawer.popoverBackground }
```

Comment:
This is being used in devmain? What's the strategy to expose those colors there?

Reply:
Thanks for catching this, this is referenced in several places in devmain. I have kept the interface as same for now and added extensions to return tokenized values.